mocks_test.go

```diff
@@ -116,3 +118,52 @@ func (m *mockCounter) With(labelValues ...string) metrics.Counter {
 	m.Called(labelValues)
 	return m
 }
+
+// mockKey is a mock for key.
```

Comment:
I added these mocks because Talaria was using resolver and key mocks from the bascule/key package. 

The new bascule version also mocks the resolver and the key so an alternative solution could be to expose the mocks in Bascule and use them in Talaria instead of adding them here. Do you have an opinion, @johnabass ?

Reply:
Added https://github.com/xmidt-org/clortho/issues/45